test/transport_web/solutions/data_reuse/using_list_test.exs

```diff
@@ -20,10 +20,10 @@ defmodule TransportWeb.Solution.DataReuse.UsingListTest do
     %_{} = ReusableData.create_dataset %{
       download_uri: "https://link.to/angers.zip",
       license: "odc-odbl",
-      title: "Angers GTFS",
+      title: "Horaires et arrêts du réseau IRIGO - format GTFS",
```

Comment:
Could we cherry-pick these changes and create an independent pull request ? I{t is my understanding these changes are tangential to his PR.

Reply:
We could, but we are in a hurry.

Reply:
OK